src/controllers/company.number.controller.ts

```diff
@@ -8,7 +8,7 @@ import {
   retrieveAccessTokenFromSession,
 } from "../services/objection.session.service";
 import logger from "../utils/logger";
-import { check, validationResult } from "express-validator/check";
+import { check, validationResult } from "express-validator";
```

Comment:
warning in console when running tests that express-validator/check was deprecated